Raptacon/Robot-2023
# Balance pid

## Types of changes


## Associated Issue(s)
Issue: <#X or issue-XX > [REMOVE THIS WHEN DONE Describe the issue you're working on here so it ties to the issue and Kanban (IMPORTANT FOR TRACKING). Make sure to create an issue if one doesn't already exist! ]

## Steps to Test
when robot is offbalanced on the charging switch, press the x on the xbox controller. The robot should autobalance. 

## Changelog

added and adjusted PID to the balancing class.

## What?

robot doesn't overshoot too much when trying to balance.

## Why?

without the pid the robot always overshoot too much, making it impossible to balance.
